### ISSUE TITLE
meson error on ubuntu 18.04

### ISSUE THREAD

**Issue by moonshaadow** [jack-mixer/jack_mixer#128]
Hi there. 
I encountered the following error trying to build on ubuntu 18.04.
I have followed the dedicated wiki, so I think I haven't forgot any step...

```
$ ninja -C builddir
ninja: Entering directory `builddir'
[4/13] Compiling C object src/jack_mix_box.p/jack_mix_box.c.o
../src/jack_mix_box.c: In function ‘usage’:
../src/jack_mix_box.c:65:2: warning: format not a string literal and no format arguments [-Wformat-security]
   65 |  printf(_usage);
      |  ^~~~~~
../src/jack_mix_box.c: In function ‘main’:
../src/jack_mix_box.c:155:3: warning: format not a string literal and no format arguments [-Wformat-security]
  155 |   fprintf(stderr, jack_mixer_error_str());
      |   ^~~~~~~
[13/13] Linking target jack_mixer/_jack_mixer.cpython-36m-x86_64-linux-gnu.so

```

**Comment by SpotlightKid** [comment]
This isn't actually an error but a but a compiler warning turned on by the use of `-Wformat-security` compiler option. I'm not sure whether this is on by default on Ubuntu or just on your system. `-Wformat-security` not included by `-Wall`, but by `-Wformat`.

The warning has no ill effects, but a Way to suppress it would be to add `-Wno-format-security` to CFLAGS. E.g.:

    CFLAGS="-Wno-format-security" meson setup -C builddir --prefix=/usr --buildtype=release
    ...

The proper fix to avoid the warning would probably be use `fputs` instead of `printf` and `fprintf`. I'll fix that in the next release.

**Comment by moonshaadow** [comment]
Thank you. I'm sorry I was unable to start `jack_mixer` and I thought it was related, but it wasn't.

In fact I had to `rm` all `jack_mixer` related files and then re-install. Now the app starts as it should be. I'm sorry I haven't kept the error logs. It was something like " unable to find app.py ". My last installation  was made before the changing of the build system to `meson`

Maybe we should add an uninstall rule and mention it in the wiki, just in case.

**Comment by SpotlightKid** [comment]
1) The error you git previously might have been caused by not specifying `--prefix=/usr` when setting up the build directory. Hard to say without knowing what you did or the error logs.

2) Un-install already exists, but you need to still have the build directory from which you ran run `meson install` / `ninja install` around. E.g. if the build directory is `builddir`, run this from the repository root:

        sudo ninja -C builddir uninstall

3) You can also run jack_mixer from the source directory for testing without installing it:

        meson setup builddir --prefix=/usr --buildtype=release
        meson compile -C builddir   # or ninja -C builddir compile
        ./tools/compile-messages.py
        ./tools/jack_mixer.sh

**Comment by moonshaadow** [comment]
Just in case someone needs it : for testing I need to replace `python` by `python3` in `jack_mixer.sh` 

Uninstall works as expected.

**Comment by SpotlightKid** [comment]
* Fixed `-Wformat-security` warnings in 81b0369.
* Fixed `jack_install.sh` script in 53916a2.
